CHANGES.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## 2.2.1
```

Comment:
Let's bump the minor version here per https://semver.org/#spec-item-7 as the PR introduces the new action param.

```suggestion
## 2.3.0
```